src/query/service/src/pipelines/processors/transforms/transform_async_function.rs

```diff
@@ -177,8 +169,23 @@ impl TransformAsyncFunction {
                             None
                         };
 
+                        let req = GetSequenceReq {
+                            ident: SequenceIdent::new(&tenant, sequence_name),
+                        };
+                        let resp = catalog.get_sequence(req, &visibility_checker).await?;
+                        let step_size = resp.meta.step as u64;
+
+                        // Calculate batch size - take the larger of count or step_size
+                        let batch_size = to_fetch.max(step_size);
```

Comment:
The implementation here seems not correct. The `step` represents the interval between two values. For example, if `step` is 2, `start` is 0, and the block's `num_rows` is 4, the sequence should be `0`, `2`, `4`, `6`. Therefore, `batch_size` should be `step * (num_rows - 1)`.

Reply:
Just keep old logic. If this is a bug should fix in other pr?

Reply:
step is the number to be increased with a meta api call.